code-pushup.config.ts

```diff
@@ -75,15 +79,38 @@ const config: CoreConfig = {
       type: 'module',
     }),
 
-    // see https://github.com/code-pushup/cli/issues/538
-    // await lighthousePlugin({
-    //   url: 'https://staging.code-pushup.dev/login',
-    //   outputPath: join('.code-pushup', LIGHTHOUSE_OUTPUT_FILE_DEFAULT),
-    //   headless: true,
-    // }),
+    await lighthousePlugin('https://codepushup.dev/', {
+      outputPath: join('.code-pushup', LIGHTHOUSE_REPORT_NAME),
```

Comment:
Why isn't this the default `outputPath` value?

Reply:
very goo point!